app/api/src/lib/discord.ts

```diff
@@ -1,19 +1,29 @@
-import {Client, Intents} from "discord.js"
+import {Client, Intents, MessageAttachment} from "discord.js"
 
 const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES]})
 
-export async function sendChat(text: string) {
+export async function sendDiscordMessage(text: string, url?: string) {
   if (!client.isReady()) {
     console.error(`Discord: client is not ready to send message ("${text}")`);
   } else {
     const channel = await client.channels.fetch(process.env.DISCORD_CHANNEL_ID);
-    channel.send(text);
+    if (url) {
+      channel.send({ embeds: [{
+        title: text,
+        image: {
+          url: url,
+        },
+      }] });
```

Comment:
Thought it would be cool to include the project image.

Reply:
Hell yeah :)